libs/service-portal/social-insurance-maintenance/src/screens/PaymentPlan/PaymentPlan.tsx

```diff
@@ -0,0 +1,293 @@
+import { useLocale, useNamespaces } from '@island.is/localization'
+import {
+  GetPaymentPlanQuery,
+  useGetPaymentPlanLazyQuery,
+} from './PaymentPlan.generated'
+import {
+  ExpandHeader,
+  amountFormat,
+  generateYears,
+} from '@island.is/service-portal/core'
+import { Problem } from '@island.is/react-spa/shared'
+import {
+  Box,
+  Text,
+  Button,
+  Table,
+  Select,
+  Stack,
+  Divider,
+  GridContainer,
+  GridRow,
+  GridColumn,
+} from '@island.is/island-ui/core'
+import { UserInfoLine, m as coreMessages } from '@island.is/service-portal/core'
+import { m } from '../../lib/messages'
+import { useMemo, useState } from 'react'
+import addYears from 'date-fns/addYears'
+import { PaymentGroupTableRow } from '../../components'
+import { PaymentPlanWrapper } from './wrapper/wrapper/PaymentPlanWrapper'
+import { SocialInsuranceMaintenancePaths } from '../../lib/paths'
+
+const PaymentPlan = () => {
+  useNamespaces('sp.social-insurance-maintenance')
+  const { formatMessage } = useLocale()
+
+  /*const [getPaymentPlanQuery, { data, loading, error }] =
+    useGetPaymentPlanLazyQuery()*/
+
+  const yearOptions = useMemo(() => {
+    const years = generateYears(addYears(new Date(), -120), new Date(), 'desc')
+    return years.map((y) => ({ value: y, label: y.toString() }))
+  }, [])
+
+  const [selectedYear, setSelectedYear] = useState(yearOptions[0])
+  /*
+  useEffect(() => {
+    getPaymentPlanQuery({
+      variables: { input: { year: selectedYear.value } },
+    })
+  }, [selectedYear, getPaymentPlanQuery])*/
```

Comment:
Remove?

Reply:
forgot to remove mock data, this is uncommented